frontend/api/answers/[id].ts

```diff
@@ -0,0 +1,21 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import { prisma } from '../../src/server/prisma';
+
+export default async function (req: VercelRequest, res: VercelResponse) {
+  const resultId = req.query.id as string;
+  if (req.method === 'GET') {
+    const result = await prisma.result.findUnique({
+      where: { id: resultId },
+      select: { id: true, value: true, createdAt: true },
+    });
+    if (result === null) {
+      return res.status(404).json({ error: `Result ${resultId} not found` });
```

Comment:
Pojďme prosím použít _[Problem details for HTTP APIs (RFC 7807)](https://tools.ietf.org/html/rfc7807)_. Tedy odpověď by měla být `application/problem+json` a vypadat něco jako:

```json
{
  "type": "https://volebnikalkulacka.cz/api/errors/result-not-found",
  "title": "Result not found",
  "detail": "Result I `${resultId}` not found.",
  "status": 404
}
```

Na té type URL zatím nic být asi nemusí, ale ať držíme tu struktura.